arangod/Aql/QueryCursor.cpp

```diff
@@ -319,17 +319,7 @@ void QueryStreamCursor::resetWakeupHandler() {
 }
 
 ExecutionState QueryStreamCursor::writeResult(VPackBuilder& builder) {
-  VPackOptions const* oldOptions = builder.options;
-  auto guard = scopeGuard([&] {
-    builder.options = oldOptions;
-  });
-  
-  VPackOptions options = VPackOptions::Defaults;
-  options.buildUnindexedArrays = true;
-  options.buildUnindexedObjects = true;
-  options.escapeUnicode = true;
```

Comment:
The `escapeUnicode` property is not set anymore after this change.
While it is quirky to set it, it was there for a reason: some parsers (namely the JSON parser in V8/node.js) was able to parse ASCII-based JSON a lot faster than JSON that contained Unicode characters.
E.g. `{"pädo": "bär" }` was (at least some time ago) a lot more expensive to parse than `{"p\uxxxxdo": "b\uxxxxr"}`, because the latter caused some internal state transition in the JSON implementation, migrating the string from single-byte characters to multi-byte characters. Not sure if still relevant, but this should be checked.

Reply:
the VPackDumper does this, escapeUnicode never has had any effect here

Reply:
Ok